config/routes.rb

```diff
@@ -0,0 +1,12 @@
+Rails.application.routes.draw do
+  devise_for :users
+  resources :categories do 
+  resources :payments
+  end
+  root "home#index"
+  get 'about_us', to: 'home#about_us'
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Defines the root path route ("/")
+  # root "articles#index"
```

Comment:
- [ ] Keeping commented lines of code in our codebase is not professional and does not make our codebase look clean. Kindly remove all commented lines of code